resources/views/toast.blade.php

```diff
@@ -1,12 +1,79 @@
-<div class="toast-{{ $type }} flex items-center {{ $class ?? '' }}">
-    <div class="flex-1">{{ $message }}</div>
-
-    @if (($wireClose ?? false) || ($alpineClose ?? false))
-        <div
-            @if ($wireClose ?? false) wire:click="{{ $wireClose }}" @endif
-            @if ($alpineClose ?? false) @click="{{ $alpineClose }}" @endif
-        >
-            @svg('close', 'toast-close')
-        </div>
+@props([
+    'type' => 'info',
+    'style' => 'regular',
+    'message' => '',
+    'wireClose' => false,
+    'alpineClose' => false,
+])
+
+@php
```

Comment:
this becomes quite messy with 50 lines of classes 🤔 why not leave it as separate styles in the `_toast.css` file and set a toast class on the parent as was done before?

Reply:
@ItsANameToo I'm just following the [Tailwind-style recommendation](https://tailwindcss.com/docs/extracting-components) that encourages avoiding custom classes wherever is possible.
2. it will result in a smaller CSS file
3. Personally think is not as messy as it looks at the first sight

But most important thing: is not really a big deal so wherever you think it best